meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -741,6 +741,8 @@ MACHINE_FIRMWARE:remove:mx6-nxp-bsp = "linux-firmware-imx-sdma-imx6q"
 MACHINE_FIRMWARE:remove:mx8-nxp-bsp = "linux-firmware-imx-sdma-imx7d"
 MACHINE_FIRMWARE:append:mx6-nxp-bsp = " firmware-imx-sdma-imx6q firmwared"
 MACHINE_FIRMWARE:append:mx8-nxp-bsp = " firmware-imx-sdma-imx7d firmwared"
+# Align wayland / xwayland versions required by the BSP
+PREFERRED_VERSION_wayland:imx-nxp-bsp ?= "1.22.0.imx"
```

Comment:
Should this patch be also submitted to meta-freescale? seems to be a bsp problem.

Reply:
Yes, I will try today, because this is only relevant with freescale master + oe kirkstone, and not really needed on more recent releases, so we will see if they are ok to fork it.

Reply:
Merged.